third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -3935,6 +3935,11 @@ def nlargest(self, n: int, columns, keep: str = "first"):
         ``df.sort_values(columns, ascending=False).head(n)``, but more
         performant.
```

Comment:
Looks like nothing gets rendered except the first line in the docstring. Does all that get fixed by this reordering? https://screenshot.googleplex.com/5fTe6kSNpdMqxXc

Reply:
I believe you're referring to the new method overview page.

I'm fixing this one: https://screenshot.googleplex.com/4VPB3N8qWLWbmjX

Reply:
Ah, you're right, thanks!